### ISSUE TITLE
Apple deprecating Darksky API

### ISSUE THREAD

**Issue by staze** [SANdood/MeteoWeather#3]
Not sure how to address this. looks like there's a cost for over 500k queries per month, and it's tied to dev id (seemingly). =/

From Apple:

Hello there,
--
We wanted to let you know about a change coming to Dark Sky. Beginning on March 31st, 2023 the Dark Sky API will no longer be available. To prepare, we encourage you to learn more about WeatherKit, a new Apple API available on iOS, iPadOS, macOS, tvOS, and web that provides access to the new Apple Weather forecast data.
You can learn more here.
As always, we appreciate your support, and will continue to update you with any future changes.
Thanks,The Dark Sky Team

Hello there,
We wanted to let you know about a change coming to Dark Sky. Beginning on March 31st, 2023 the Dark Sky API will no longer be available. To prepare, we encourage you to learn more about WeatherKit, a new Apple API available on iOS, iPadOS, macOS, tvOS, and web that provides access to the new Apple Weather forecast data.
You can learn more [here](https://developer.apple.com/weatherkit/).
As always, we appreciate your support, and will continue to update you with any future changes.

Thanks,
The Dark Sky Team

**Comment by SANdood** [comment]
Yup - looks like MeteoWeather will only be able to use the (limited) forecast data provided by the MeteoBridge API...

**Comment by staze** [comment]
yup... could possibly use: https://openweathermap.org/api

Though would have to limit frequency (only 1000/day is free). Might be others... super unclear.

**Comment by staze** [comment]
Howdy sir! 

Since then, I have flipped my Meteobridge integration over to using Meteobridge for forecast, but the device still throws the error

"TWC weather data is not available on Hubitat - please configure DarkSky or MeteoBridge for forecast data"

Does that selector not work to pick Meteobridge? 

Interestingly, I see forecast data on the device. So that error seems to either be a false flag, or... something?
